.github/workflows/publish-sphinx.yml

```diff
@@ -77,14 +79,15 @@ jobs:
         wget https://github.com/nexusformat/code/raw/master/applications/NXtranslate/docs/NXtranslate.pdf -O ./build/manual/source/_static/NXtranslate.pdf
 
     - name: Build (html) and Commit
-      uses: sphinx-notes/pages@master
+      uses: sphinx-notes/pages@v2
       with:
         # path to conf.py directory
         documentation_path: build/manual/source
+        target_path: ..
```

Comment:
@prjemian This should probably be `target_path: '..'`  (with the quotes)

Reply:
? What is target_path: '..' supposed to achieve? Are you trying to publish to https://nexusformat.github.io/ instead of https://nexusformat.github.io/definitions/?

Right, that was the intent.

Change in the YAML not necessary.  Quotes are need to avoid using YAML syntax characters, such as `!`, `#`, etc.
 The actions workflow failed later, with this error:

```
  Error: Invalid status code: 128
  '/tmp/pages-Db29kn5xrc/./_static/NeXusIntern.pdf' -> '.././_static/NeXusIntern.pdf'
      at ChildProcess.<anonymous> (/home/runner/work/_actions/sphinx-notes/pages/v2/main.js:9:19)
      at ChildProcess.emit (events.js:314:20)
      at maybeClose (internal/child_process.js:1022:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5) {
  '/tmp/pages-Db29kn5xrc/./_static/nxdl_vocabulary.yml' -> '.././_static/nxdl_vocabulary.yml'
```

I was guessing that `target_path: ..` (publishing to the parent directory) would work.  Seems that is not in the scope of this Github Action.